api/hooks/sdtdLogs/LoggingObject.js

```diff
@@ -73,7 +73,7 @@ class LoggingObject extends EventEmitter {
     const job = await this.queue.getJob(jobId);
 
     // eslint-disable-next-line eqeqeq
-    if (job.data.serverId != this.serverId) {
+    if (job.data != this.data) {
```

Comment:
The problem is that `job` is null. So if the code tries to access `job.data` it errors out.

Your patch does not fix this! What we need is that IF `job` is not defined, we `return` early and do not execute the rest of the function